src/NServiceBus.Testing.Tests/Sagas/BasicSaga.cs

```diff
@@ -0,0 +1,99 @@
+﻿namespace NServiceBus.Testing.Tests.Sagas
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class BasicSaga
+    {
+        [Test]
+        public async Task TestBasicSaga()
+        {
+            var testableSaga = new TestableSaga<ShippingPolicy, ShippingPolicyData>();
+
+            var placeResult = await testableSaga.Process(new OrderPlaced { OrderId = "abc" });
```

Comment:
Would there be any merit in having a `.Start` overload to make things more explicit? 

Disclaimer: I know I have been on record saying that "pretty much all messages should be allowed to start a saga" but as long as we have `IAmStartedBy...` this type of feature might still make sense

Reply:
I get what you're trying to say but in the context of testing, the saga is a thing you throw a bunch of messages at and hope the right things pop out, so differentiating the methods used to throw the messages at the saga sounds odd to me.